src/main/java/com/artozersky/HackerNewsAPI/dto/PostResponseDTO.java

```diff
@@ -80,4 +85,21 @@ public void setScore(Double score) {
     public void setMessage(String message) {
         this.message = message;
     }
+    @Override
+    public LocalDateTime getCreatedAt() {
+        return createdAt;
+    }
+    @Override
+    public void setCreatedAt(LocalDateTime createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    // Calculated dynamically
+    @Override
```

Comment:
Why should you calculate it here and not in the schema? the schema in the response will do it automatically and then the model mapper will map to the relevant fields. please share your thoughts.

Reply:
this function was never used